### ISSUE TITLE
Warnings should be issued when workbook-pages.txt contains pages that are not referred to in the lesson plan

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#981]
The algebra workbook contains two pages that we had removed from the lesson plan but forgotten to remove from workbook-pages.txt and no warning was being issued. This is a regression.

**Comment by schanzer** [comment]
@ds26gte note that the workbook-pages.txt file should be a subset of the union of all @printable-exercises and @opt-printable-exercises. There may be opt-printables that aren't in the workbook, but all printables should be.

**Comment by ds26gte** [comment]
The workbook page `whats-on-your-mind.adoc` is present in many lessons and just contains a sequence of blank lines. These are not referred to via `@printable-exercise` in the lesson plan. There may be other boilerplate pages like this.

**Comment by schanzer** [comment]
@flannery-denny I did some testing tonight, and it looks like Dorai's fix is working nicely! Please close if you're happy with it.

**Comment by flannery-denny** [comment]
@schanzer I don't know how to test this without breaking things, so I'm going to assume you've investigated and clpse without further review since it's not a public-facing issue.

**Comment by flannery-denny** [comment]
I just discovered that the Defining-Values lesson was missing the page `which-values-to-define`. It was listed in workbook-pages.txt and i'm pretty sure it was in the printed copy of the workbook I reviewed, but the lesson never mentioned it. 

Instead of giving me a warning when I built the lesson, the build system just didn't include it in the materials section of the table, so it was undiscoverable.
